GUI/XLSearchTaskWindow.xaml

```diff
@@ -63,36 +63,33 @@
                                     <TextBox x:Name="txtXLBetaPrecusorMsTl" Width="45" IsEnabled="{Binding IsChecked, ElementName=ckbSearchWithXLAllBeta}" />
                                     <ComboBox x:Name="cbbXLBetaprecusorMsTl" IsEnabled="{Binding IsChecked, ElementName=ckbSearchWithXLAllBeta}" />
                                 </StackPanel>-->
-                                <StackPanel Orientation="Horizontal" Grid.Row="1" Grid.Column="0">
-                                    <Label x:Name="lblXLprecusorMsTl"  Content="Crosslink Precusor mass tolerance" />
+                                <StackPanel Orientation="Horizontal" Grid.Row="1" Grid.Column="0" Margin="5">
```

Comment:
I think you're not synced up or something, these are changes I made to the XL Search Task window recently